tests/utils/helpers.ts

```diff
@@ -0,0 +1,54 @@
+import { Build, BuildStatus, VisualApi } from '@saucelabs/visual';
+import { randomBytes } from 'crypto';
+
+export const RE_VISUAL_BUILD_LINK =
+  /https:\/\/app\.(([a-z0-9-]+)\.)?saucelabs\.(com|net)\/visual\/builds\/([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})/g;
+export const RE_VISUAL_BUILD_ID =
+  /([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})/g;
+
+export const runIdOrRandomId = process.env.GITHUB_RUN_ID
+  ? process.env.GITHUB_RUN_ID + process.env.GITHUB_RUN_ATTEMPT
```

Comment:
I'd suggest to explicitely say it comes from GH:
Something like:
```suggestion
  ? `Github: ${process.env.GITHUB_RUN_ID}${process.env.GITHUB_RUN_ATTEMPT}`
```